src/components/MovieCard/MovieCard.js

```diff
@@ -1,42 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import ThreeDotsMenu from '../ThreeDotsMenu/ThreeDotsMenu';
+import { getYearFromDate } from '../../utils/utils';
+import { useHover } from '../../hooks/hooks';
 import './MovieCard.scss';
 
 export default function MovieCard(props) {
   const {
-    title,
-    description,
-    year,
-    poster,
+    movie,
+    onEdit,
+    onDelete,
+    onClick,
   } = props;
 
-  const handleIconClick = () => {
-  };
+  const [hoverRef, isHovered] = useHover();
```

Comment:
Please use useRef hook for refs

Reply:
`useRef` is used inside `useHover`. Is this incorrect?
Do we need to use `useRef` in MovieCard and pass it as parameter to `useHover` similar to `useOutsideClick`?